source/manual/rename-a-country.html.md

```diff
@@ -127,24 +127,20 @@ Failing this, there is a [rake task](https://github.com/alphagov/search-api/blob
 
 ### 7. Update Specialist Publisher
 
-1. Create a content schemas [pull request](https://github.com/alphagov/govuk-content-schemas/pull/1014) to support both countries temporarily (around the data migration).
+1. Create and deploy pull requests for GOV.UK Content Schemas ([example](https://github.com/alphagov/govuk-content-schemas/pull/1014)) and Specialist Publisher ([example](https://github.com/alphagov/specialist-publisher/pull/1722/commits/79c10d173f8294fef25b07678a7e74213e78e424)) to support both countries temporarily (around the data migration).
 
-1. Create a [pull request](https://github.com/alphagov/specialist-publisher/pull/1722/commits/79c10d173f8294fef25b07678a7e74213e78e424) to support both countries temporarily (around the data migration).
+1. Run the rake task [publishing_api:publish_finder[finder]](https://deploy.integration.publishing.service.gov.uk/job/run-rake-task/parambuild/?TARGET_APPLICATION=specialist-publisher&MACHINE_CLASS=backend&RAKE_TASK=publishing_api:publish_finder[finder]) to update the finder options for:
+  - `export_health_certificates`
+  - `international_development_funds`
```

Comment:
This nested list seems to throw Github's markdown rendering off for the ordered list it is within: https://github.com/alphagov/govuk-developer-docs/blob/035b20950f5cdf40cbd963665090fb390c6eaa09/source/manual/rename-a-country.html.md#7-update-specialist-publisher it resets it back to "1." for the subsequent item. Since it's only 2 items you might just want to show them inline? 

Interestingly this doesn't seem to be a problem with the nested lists earlier in this document where they've gone for an incremental ordering as opposed to listing all as "1."

Reply:
Also since a step later says "Re-publish the finders, as above" it might be easier to understand if this used the terminology of re-publish the finders. For example: 

> Run the rake task publishing_api:publish_finder\[finder\] to re-publish these finders:

Reply:
Sure, I've revised that sentence. Thanks for spotting the numbering; that's quite frustrating and worked properly locally - I've switched it to manual.

Reply:
Dah, this thing is so needy! I thought that's just the way it rendered them - I've forced pushed a fix for the other bullets in the doc as well.